src/Repository/ProductOptionRepository.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file has been created by developers from BitBag.
+ * Feel free to contact us once you face any issues or want to start
+ * another great project.
+ * You can find more information about us on https://bitbag.io and write us
+ * an email on hello@bitbag.io.
+ */
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusElasticsearchPlugin\Repository;
+
+use Doctrine\ORM\QueryBuilder;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+
+class ProductOptionRepository implements ProductOptionRepositoryInterface
```

Comment:
final

Reply:
Repositories and entities in public projects should not (and cannot) be defined as final

Point 7 of BitBag Bible

Reply:
You're right, I based this comment on other repositories in this plugin, so please remove final from other repositories to be consistent.

Reply:
Sure, done!